README.md

```diff
@@ -368,34 +361,55 @@ When implementing the `map*Dispatch` methods, apps can create and dispatch custo
         }),
     };
 
-As can be seen in this example, when passing a function to `dispatch`, it will receive the `api` object as the third parameter. Here we pull out the `logger`, and use that to add a log message.
-
-The properties provided by the `api` object are described below.
+#### Hosted modules / internal api
 
 <table>
   <tbody>
     <tr>
-      <th>Property</th>
+      <th>Path</th>
+      <th>Exported</th>
```

Comment:
Is the `Exported` column really necessary? As long as we link to the library documentation, the developer should have what he/she needs to import and use the library.